CHANGELOG.md

```diff
@@ -9,7 +9,11 @@ All notable changes to the Zowe Client Python SDK will be documented in this fil
 - Added logger class to core SDK [#185](https://github.com/zowe/zowe-client-python-sdk/issues/185)
 - Added classes for handling `Datasets`, `USSFiles`, and `FileSystems` in favor of the single Files class. [#264](https://github.com/zowe/zowe-client-python-sdk/issues/264)
 - Refactored tests into proper folders and files [#265](https://github.com/zowe/zowe-client-python-sdk/issues/265)
-- Fix the bug on `upload_file_to_dsn`. [#104](https://github.com/zowe/zowe-client-python-sdk/issues/104)
+- Fixed the bug on `upload_file_to_dsn` [#104](https://github.com/zowe/zowe-client-python-sdk/issues/104)
```

Comment:
Sorry (again) for requesting changes but could we expand a bit on this entry?
That way people don't have to click the issue to see what the bug was about.

```suggestion
- Fixed a bug on `upload_file_to_dsn` where it would not properly convert line endings on Windows. [#104](https://github.com/zowe/zowe-client-python-sdk/issues/104)
```

Reply:
Since this line mentions a bug fix, it should be moved to the "Bug Fixes" section of the changelog 🙂